src/mon/FSCommands.cc

```diff
@@ -361,12 +361,6 @@ class SetHandler : public FileSystemCommandHandler
         });
 	ss << "disabled new snapshots";
       } else {
-	string confirm;
-	if (!cmd_getval(g_ceph_context, cmdmap, "confirm", confirm) ||
-	    confirm != "--yes-i-really-mean-it") {
```

Comment:
this breaks https://github.com/ceph/ceph/blob/master/qa/workunits/cephtool/test.sh#L991